editor/js/reaction.js

```diff
@@ -65,7 +65,7 @@ async function init(fileName, index) {
   loadReaction(reaction);
   clean();
   // Trigger reaction-level validation.
-  setTimeout(validateReaction, 1000);
```

Comment:
Why is this removed? I thought there was some startup lag that this was intended to control for?

Reply:
Didn't solve the problem we were originally trying to solve... I figured the timeout wasn't necessary. 

After further thought, even though I can't notice any issues in my own testing, I think the timeout is still safer (perhaps if startup takes a while for example). I'll put it back.

Reply:
After even _more_ testing, validateReaction should be okay without a timeout. Loading, adding, and starting to show the validation divs all take place before validateReaction is called. Even though we have a "show" transition, the divs are still visible before the transition actually finishes.

I'll stick with no timeout for now...